src/onboarding.ts

```diff
@@ -228,14 +229,66 @@ promptfoo eval
 Afterwards, you can view the results by running \`promptfoo view\`
 `;
 
-function recordOnboardingStep(step: string, properties: Record<string, EventValue> = {}) {
+function recordOnboardingStep(step: string, properties: any = {}) {
   telemetry.recordAndSend('funnel', {
     type: 'eval onboarding',
     step,
     ...properties,
   });
 }
 
+/**
+ * Iterate through user choices and determine if the user has selected a provider that needs an API key
+ * but has not set and API key in their enviorment.
```

Comment:
"has not set _an_ API key"

Reply:
Updated and I also fixed my misspell of _environment_